data/assets/scene/solarsystem/missions/gaia/dashboard.asset

```diff
@@ -1,3 +1,7 @@
+asset.require("./gaia")
```

Comment:
I added it because the dashboard.asset will throw an error when loading it without loading the gaia model first.  I changed the file to make the dependency more clear.

OBS that it only includes the Gaia model, not the gaia stars

Reply:
Ah, I see! Makes sense